src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -310,6 +325,18 @@ public void Stop()
         }
 
         #region private 
+        private void NotifyError(Exception exception)
+        {
+            // Throw and catch the exception.  This lets it be seen by the debugger
+            // ETW, and other monitoring tools.   However we immediately swallow the
+            // exception.   We may wish in the future to allow users to hook this 
+            // in other useful ways but for now we simply swallow the exceptions.  
+            try
+            {
+                throw exception;
```

Comment:
It looks like all of the exceptions passed in here are created at the call site, true?  If it's not true, it'd be good to use ExceptionDispatchInfo.Throw (or EDI.Capture(...).Throw if this needs to sit on top of netstandard) to avoid overwriting the existing stack trace, so that it'd be visible in the debugger.